pylearn2/scripts/papers/maxout/compute_test_err.py

```diff
@@ -51,8 +51,6 @@ def accs():
                             data_specs = model.cost_from_X_data_specs())
     for item in iterator:
         x_arg, y_arg = item
-        if Xb.ndim > 2:
-            x_arg = test.get_topological_view(x_arg)
```

Comment:
Was it because x_arg used to be 2D no matter what ```model.get_input_space().make_batch_theano()``` shape was? Where is the conversion done in the iterator?